invideoquiz/public/js/src/invideoquiz.js

```diff
@@ -102,8 +112,10 @@ function InVideoQuizXBlock(runtime, element) {
         var problemToDisplay;
 
         video.on('play', function () {
-          videoState = videoState || video.data('video-player-state');
-
+          if (videoType !== 'video_xblock'){
```

Comment:
Consider moving 'video_xblock' string to a variable(constant). The reason is simple - you use the case three times as I can observe from your PR. It is to ease to make a mistake (type) in one of the case and your logic will be suffer. Or you want to redo and change `video_xblock` to whatever else and you must edit all cases. It is bad practice.